DigitalLearningSolutions.Web/ViewModels/LearningMenu/TutorialViewModel.cs

```diff
@@ -35,5 +40,21 @@ private static string ParseMinutes(int minutes)
             }
             return $"{minutes}m";
         }
+
+        private static string? ParseObjectives(string? objectives)
+        {
+            if (objectives == null)
+            {
+                return null;
+            }
+
+            var doc = new HtmlDocument();
+            doc.LoadHtml(objectives);
+
+            var body = doc.DocumentNode.SelectNodes("//body");
+
+            return body?.FirstOrDefault()?.InnerHtml
+                   ?? WebUtility.HtmlDecode(objectives);

```

Comment:
Sorry, why do we need to call `HtmlDecode` here? 🙂

Reply:
I'm not sure if I need this actually. It converts HTTP strings into normal character strings (eg. from `&gt;` to `>`) but now that I think of it I doubt it will be stored like that in the database (and I can't find any examples). Removed in 3ff95e1 🙂